src/core/configuration.ts

```diff
@@ -231,6 +236,25 @@ namespace Configuration {
     mode: Configuration.RecommendationMode;
   }
 
+  export interface RecommendationsINav {
```

Comment:
recommendations is getting large enough that we should use a `namespace` to manage it,
look for examples in the rest of this file as to how to namespace interfaces (rather than prefixing the names with `Recommendations`)